diff --git a/adguardhome/Makefile b/adguardhome/Makefile
index a5c4d67..528fc4a 100644
--- a/Makefile
+++ b/Makefile
@@ -62,14 +62,5 @@ define Build/Compile
 	)
 endef
 
-define Package/adguardhome/install
-	$(call GoPackage/Package/Install/Bin,$(1))
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/adguardhome.init $(1)/etc/init.d/adguardhome
-
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_DATA) ./files/adguardhome.config $(1)/etc/config/adguardhome
-endef
-
 $(eval $(call GoBinPackage,adguardhome))
 $(eval $(call BuildPackage,adguardhome))
